### PR TITLE
v1.19.1 (release)

### PR DESCRIPTION
**Pull request by caass**

### CODE REVIEW

**Review by threepointone — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -562,7 +667,7 @@
 
     Y'all, we messed up and applied the wrong fix. The change which caused this problem was [this](https://github.com/cloudflare/wrangler/pull/1566).
 
-    [ObsidianMinor]: https://github.com/ObsidianMinor
+    [obsidianminor]: https://github.com/ObsidianMinor
```
Comment (jyn514) [inline CHANGELOG.md:670]: Did we change the tool that generates these or something? Seems weird it's changing the capitalization. Markdown is case insensitive though, so this should still work.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2142,17 +2249,17 @@
...
+  Wrangler orchestrates a few other tools under the hood, notably [`wasm-pack`](https://github.com/rustwasm/wasm-pack) and [`cargo-generate`](https://github.com/ashleygwilliams/cargo-generate). We use a library called [`binary-install`](https://github.com/rustwasm/binary-install) to fetch and cache binaries we download. However, to avoid downloading unnecessarily, we first check if the user has a copy locally on their machine that they had `cargo install`'d. We had a bug where in this logic branch, we _didn't_ check that the local version was the most up-to-date version. This meant that users who had an older installed version may run into errors when wrangler expected to use features of a newer version of that tool. This PR adds the logic to check for the version and will install and cache a newer version for wrangler to use (leaving your local version as is!).
```
Comment (jyn514) [inline CHANGELOG.md:2252]: ditto, this spacing change is weird. How are these generated?

File: README.md [hunk 3]
```diff
@@ -1,191 +0,0 @@
-# 🤠 wrangler
```
Comment (jyn514) [inline README.md:1]: I don't think you meant to delete the readme :P

**Review by jyn514 — CHANGES_REQUESTED** [review]
Deleting the readme definitely seems wrong. The rest I don't think needs to block the release.

**Review by jyn514 — APPROVED** [review]

**Pull request merged**